### PR TITLE
More reasonable and restrained cuts to Section 3 of the introduction

### PR DESCRIPTION
**Pull request by pavpanchekha**
This section is pretty powerful so I didn't want to make any big changes that could hurt that. But in a few places I thought it could be shortened.

One big question I have about this section is the last paragraph. I think this paragraph is what my kind of dopey trailer in #55 was trying to be. What do you think of moving this section ("The browser and me") before the previous one ("Explaining the black box")? And then using the last paragraph of this section as the last paragraph of "Explaining the black box"?

### CODE REVIEW

File: book/browsersandweb.md
Comment (chrishtr) [inline book/browsersandweb.md:191]:
```suggestion
unimaginably complex. And everywhere you look, you see the evolution and history of
```

File: book/browsersandweb.md
Comment (chrishtr) [inline book/browsersandweb.md:196]: I think I like retaining "and" here.

File: book/browsersandweb.md
Comment (chrishtr) [inline book/browsersandweb.md:245]:
```suggestion
browsers are so important that we browser developers can afford an
```

**Comment by pavpanchekha** [comment]
You're right, the "and" works well. I'll let you merge this if you like it.

**Review by chrishtr — APPROVED** [review]

**Pull request merged**
